### DISCUSSION TITLE
Return/Go back after clicking a reference link

### DISCUSSION THREAD

**Discussion by aksajja** [James-Yu/LaTeX-Workshop#3105] [Q&A]
## Preliminary questions 
In the pdf view, the 'Return'/'Go back' command doesn't work for internal labels referenced with \ref and \hyperref commands.

### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you can not see the requested feature. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before requesting a feature.

You visited the wiki?: Yes

## Is your requested feature  related to a problem? Please describe. [Required]
It is not a problem but a good to have feature.

## To Reproduce  [Required]

1. Create a \label{abcd} and reference it with \ref{abcd} or \hyperref{abcd}
2. Click on the reference link.
3. Try to do the equivalent of 'go back one page' in a browser like Mozilla Firefox. 

## Describe the solution you'd like  [Required]

After clicking and being taken to a reference, return to point of reference call on pressing 'return/go back'.

## Logs [Required]

N/A

### LaTeX Workshop Output [Required]

N/A

### Developer Tools Console [Required]

N/A

## Desktop [Required]

 - OS: Linux x64 5.15.12-arch1-1
 - VS Code version: 1.63.2
 - Extension version: v8.23.0
 - TeX distribution version: TeX 3.141592653 (TeX Live 2021/Arch Linux)

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.

**Comment by tamuratak** [comment]
No response.

**Comment by James-Yu** [comment]
The back and forward buttons on the top bar of viewer work as you suggested.
